examples/editor/index.js

```diff
@@ -365,3 +369,101 @@ document.getElementById('fullscreen').onclick = () => {
     document.querySelector('.map-info').style.display = 'none';
     map.resize();
 };
+
+const $exportMapButton = document.getElementById('export-map-button');
+const $copyHTMLButton = document.getElementById('copy-html-button');
+const $mapTextarea = document.getElementById('map-textarea');
+
+$exportMapButton.addEventListener('click', () => {
+    const config = getJSONConfig();
+    $mapTextarea.value = generateSnippet(config);
+});
+
+$copyHTMLButton.addEventListener('click', () => {
+    $mapTextarea.select();
+    document.execCommand('copy');
+});
+
+
+/**
+
+
+"a":"sf_stclines",
```

Comment:
What is this? :joy:

Reply:
do you mean the comment? hahha . is to keep track of the config format

Reply:
ah, ok, lol